### PR TITLE
feat : added badge page

### PR DESCRIPTION
**Pull request by iamalhera**

### CODE REVIEW

File: src/components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.badge-primary{
+    position : absolute ;
+    background-color : var(--ca-primary-color-lighter);
+    color : var(--white-color);
+    padding : 0.6rem ;
+    border-radius : 2rem ;
+    top : 10px ;
+    left : 610px ;
```
Comment (rahulk31) [inline src/components/badge/badge.css]: You should work with these values as the badges flow toward right when the text content is lesser or overflows when text content is more. May be % values can help here.

File: src/components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+.badge-red{
+    position : absolute ;
+    background-color : red;
+    padding : 0.6rem ;
+    color : var(--white-color);
+    border-radius : 2rem ;
+    top : 3px ;
+    left : 340px ;
```
Comment (rahulk31) [inline src/components/badge/badge.css]: Same here.

File: src/components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+.badge-light{
+    position : absolute ;
+    background-color : var(--ca-light-bg);
+    padding : 0.5rem ;
+    border : 1px solid black   ;
+    border-radius : 2rem ;
+    top : 2px ;
+    left : 295px ;
```
Comment (rahulk31) [inline src/components/badge/badge.css]: and here as well

**Review by rahulk31 — COMMENTED** [review]
Overall code looks good to me and the code is neat and readable. Great Work.

**Review by sri-rishi — COMMENTED** [review]
Add badges on icon as mentioned in LMS

File: src/components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    background-color: var(--status-away);
+    right: 1rem;
+    bottom : 1rem;
+}
+
+.badge-primary{
+    position : absolute ;
+    background-color : var(--ca-primary-color-lighter);
+    color : var(--white-color);
+    padding : 0.6rem ;
```
Comment (sri-rishi) [inline src/components/badge/badge.css]: use px for units less than 1rem

**Review by sri-rishi — CHANGES_REQUESTED** [review]

File: src/components/badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
+@import url('/src/components/utils.css');
+@import url('/src/components/avatar/avatar.css');
+
+.badge-container{
+    position : relative ;
+}
+.badge-status {
+    position: absolute;
+    min-width: 1.8rem;
```
Comment (veereshbv04) [inline src/components/badge/badge.css]: Badge on avatars look very small, increase its size

Comment (iamalhera) [reply src/components/badge/badge.css]: thanks for the suggestion

**Pull request merged**